db/migrate/20201101204435_change_enabled_name.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeEnabledName < ActiveRecord::Migration[5.2]
+  def change
+    rename_column :merchants, :enabled, :enabled?
```

Comment:
Did you know that if you had left the name as `enabled`, rails gives you the `enabled?` method for free? That is, you wouldn't need to rename the column via a migration, rails anticipates you would want to query a boolean and gives you ? methods!